TinyLoRa.h

```diff
@@ -69,7 +69,9 @@ typedef enum rfm_datarates
 } rfm_datarates_t;
 
 /** Region configuration*/
-#define US902 ///< Used in USA, Canada and South America
+#if !defined(EU863) && !defined(AU915) && !defined(AS920)  
+  #define US902 ///< Used in USA, Canada and South America
+#endif
```

Comment:
@nerdyscout  I like the idea of exposing the region configuration from the sketch:
https://github.com/adafruit/TinyLoRa/blob/master/examples/hello_LoRa/hello_LoRa.ino#L14. I'd like to go with a different implementation, but...

Could you comment with an example sketch to show me how you're using it from outside the US (I'm based in the US)?